main/src/cgeo/geocaching/Geocache.java

```diff
@@ -958,15 +955,116 @@ public void setMyVote(final float myVote) {
         this.myVote = myVote;
     }
 
+    /**
+     * Get the current inventory count
+     *
+     * @return the inventory size
+     */
     public int getInventoryItems() {
         return inventoryItems;
     }
 
+    /**
+     * Set the current inventory count
+     *
+     * @param inventoryItems the new inventory size
+     */
     public void setInventoryItems(final int inventoryItems) {
         this.inventoryItems = inventoryItems;
     }
 
     /**
+     * Add new item number to the current inventory
+     * Warning! It is impossible to prevent duplicates. use with caution!
+     * Prefer the use of mergeInventory.
+     *
+     * @param inventoryItems the new inventory size to add
+     */
+    public void mergeInventoryItems(final int inventoryItems) {
```

Comment:
If I get it right, this method has no caller (besides a test), and should therefore be removed.